src/app/requestService/approveServiceRequest.js

```diff
@@ -215,7 +219,7 @@ const post = async (req, res) => {
     viewModel.service.roles.map((i) => i.name),
     mngUserOrgPermission,
   );
-
+  const bannerTing = await createUserBanners(req.user.id, -3);
```

Comment:
do you mean bannerTag? can you please explain in comments what is -3

Reply:
removed